hicommonwealth/commonwealth
# Create Community Backend Refactor

Refactored the error handling and added support for new create community. Made it backwards compatible with old create community.

## Link to Issue
Closes: #5799

## Description of Changes
- Refactored out create_community error handling in zod
- Adds error handling for fields that don't yet exist, but will need to be added.

## Test Plan
- go to /createCommunity. Create a successful community, make sure it passes.
- Try adding in bad data to get errors. For example add an image url that is not a url and try to submit, make sure it fails
